internal/database/mysql.go

```diff
@@ -138,16 +141,23 @@ func (db *MySQLDatabase) getOpenID() int {
 }
 
 func (db *MySQLDatabase) AddUser(username string, email string, password string) error {
+	hashedPassword, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
+	log.Printf("Hashed Password: %s, from User Password: %s", hashedPassword, password)
+
+	if err != nil {
+		log.Println("Error hashing password -", err)
+		return err
+	}
 	var (
```

Comment:
All variable declarations in this format should be at the top of the fucntion, wither break them up into separate variables with := or move to the top.